AppCenterDataStorage/AppCenterDataStorage/MSDataStore.h

```diff
@@ -169,9 +169,9 @@ typedef void (^MSPaginatedDocumentsCompletionHandler)(MSPaginatedDocuments<T> *d
  * @param documentId The CosmosDB document id.
  * @param completionHandler Callback to accept any errors.
  */
-+ (void)deleteDocumentWithPartition:(NSString *)partition
-                         documentId:(NSString *)documentId
-                  completionHandler:(MSDocumentWrapperCompletionHandler)completionHandler;
++ (void)deleteWithPartition:(NSString *)partition
```

Comment:
Did we get approval to change the public API names? What is the purpose of this and how does it relate to this PR?

Reply:
I approved it using my 👑powers. More seriously it was a typo on iOS. No other public APIs were/are using "document" in the name. I fixed the [specs](https://msmobilecenter.visualstudio.com/Mobile-Center/_wiki/wikis/Mobile-Center.wiki?wikiVersion=GBwikiMaster&pagePath=%2FHome%2FTeams%2FMBaaS%2FData%2FSpecifications%2FiOS%20SDK%20Signatures&pageId=1975&_a=compare&version=9a4481cb27cb1ba9b6b2b6dcd618804284282616) while at it.